dot-studio/svc_coord

```diff
@@ -33,23 +33,23 @@ document "wait_for_success" <<DOC
   TIMEOUT=60 wait_for_success curl localhost:8080/status
 DOC
 function wait_for_success() {
-  local SECONDS_WAITING=${TIMEOUT:-60};
-  local COUNTER=0;
+  local SECONDS_WAITING=${TIMEOUT:-60}
+  local COUNTER=0
 
   while : ; do
-    if eval "$@" >/dev/null 2>/dev/null
-    then
+    if eval "$@" >/dev/null 2>/dev/null; then
```

Comment:
As with other scripts, I'm skeptical of the need for `eval` here. Also, why not
```bash
  while "$@" &> /dev/null; do
    sleep 1
```
?